tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferences/BodyModificationsPage.tsx

```diff
@@ -0,0 +1,311 @@
+import { useState } from 'react';
+import {
+  Box,
+  Button,
+  Divider,
+  Icon,
+  Modal,
+  Section,
+  Table,
+} from 'tgui-core/components';
+
+import { useBackend } from '../../../backend';
+import { LoadingScreen } from '../../common/LoadingScreen';
+import { useServerPrefs } from '../useServerPrefs';
+import { BodyModification, PreferencesMenuData } from './data';
+
+type BodyModificationsProps = {
+  handleClose: () => void;
+};
+
+export const BodyModificationsPage = (props: BodyModificationsProps) => {
+  const serverData = useServerPrefs();
+  if (!serverData) {
+    return <LoadingScreen />;
+  }
+
+  return (
+    <Modal
+      style={{
+        width: '400px',
+        'background-color': 'transparent',
+        'max-width': '40vw',
```

Comment:
```suggestion
        backgroundColor: 'transparent',
        maxWidth: '40vw',
```
Замени подобные использования на предложенные выше
В реакте не используется CSS формат в стилях, используется camelCase